app/models/vaccination_record.rb

```diff
@@ -166,6 +167,8 @@ class VaccinationRecord < ApplicationRecord
               less_than_or_equal_to: -> { Time.current }
             }
 
+  before_create :set_notify_parents_value
```

Comment:
Generally we try to avoid using callbacks, it can lead to unexpected side-effects, and they're also not run when importing records in bulk. It also means that it's now not possible to create a vaccination record without this callback being run.

I would suggest instead that we run the equivalent of this [when the vaccination records are first created](https://github.com/nhsuk/manage-vaccinations-in-schools/blob/de3864d08401930a30d17dfb19f6fa5833a8aa68/app/controllers/draft_vaccination_records_controller.rb#L104), which also means we already have access to the consents at the time, the academic year, etc.